chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java

```diff
@@ -441,6 +427,11 @@ private Surveillance convertToDomain(final PendingSurveillanceEntity pr) {
                 reqType.setName(preq.getRequirementType());
                 req.setType(reqType);
 
+                CertifiedProduct cp = new CertifiedProduct();
+                cp.setId(pr.getCertifiedProductId());
+                cp.setChplProductNumber(pr.getCertifiedProductUniqueId());
```

Comment:
Did you test these in the UI? I don't remember what all of the fields are that are required there, but the UI can't lose functionality based on not having the detailed information

Reply:
Yes -  I tested the UI.  I uploaded a spreadheet, rejected and approved pending surveillances, and did some basic clicking around.  Seems to work w/o issues.